### ISSUE TITLE
Undefined name: OTCPreprocessing in animalai_lib.py

### ISSUE THREAD

**Issue by cclauss** [beyretb/AnimalAI-Olympics#36]
__OTCPreprocessing__ is an _undefined name_ in [this context](https://github.com/beyretb/AnimalAI-Olympics/blob/master/examples/animalai_train/animalai_train/dopamine/animalai_lib.py#L53).  Should this be [__AAIPreprocessing__](https://github.com/beyretb/AnimalAI-Olympics/blob/master/examples/animalai_train/animalai_train/dopamine/animalai_lib.py#L177-L178) instead?

[flake8](http://flake8.pycqa.org) testing of https://github.com/beyretb/AnimalAI-Olympics on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./examples/animalai_train/animalai_train/dopamine/animalai_lib.py:53:11: F821 undefined name 'OTCPreprocessing'
    env = OTCPreprocessing(env)
          ^
1     F821 undefined name 'OTCPreprocessing'
1
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

**Comment by beyretb** [comment]
Thank you for reporting this issue, it's basically an artefact of adapting the obstacle tower code to the Animal AI. I'll remove that bit and also clean this part of the code in the next release.

**Comment by beyretb** [comment]
Closing as this has been fixed in v1.0.3, thanks
